tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'm honestly not sure why `tsconfig.json` changed. Did `dompurify` update it after I installed it for this repository?

Reply:
Try rebasing against master. This should fix this. `tsconfig.json` was removed recently:

```
git pull --all
git rebase master
```

Reply:
It doesn't look like my local files changed. I also think I fetched and pulled the latest changes before starting work on this new branch, but I might be wrong.

![Screen Shot 2020-11-09 at 3 19 54 PM](https://user-images.githubusercontent.com/43425812/98607947-068d4c80-229f-11eb-896e-985220315b6c.png)

Reply:
It looks like we removed `tsconfig.json` in 417337d519d1adea5d1fb47789bec385b5116486 approximately three hours ago from the time of this writing.

Reply:
There is no `tsconfig.json` in master branch. See https://github.com/amilajack/freesound-player

Reply:
It looks like the changes from this pull request are "on top of" the latest changes, including commit 417337d519d1adea5d1fb47789bec385b5116486 where we deleted `tsconfig.json`. I'm not sure why `tsconfig.json` is back, but should I just remove it and push that change here?
 
![Screen Shot 2020-11-09 at 3 42 59 PM](https://user-images.githubusercontent.com/43425812/98609506-40138700-22a2-11eb-918c-d96d898eea15.png)